flocker/node/agents/test/test_blockdevice.py

```diff
@@ -441,10 +444,12 @@ def test_one_device(self):
         and the volume's filesystem is mounted in the right place.
         """
         dataset_id = uuid4()
+        requested_size = REALISTIC_BLOCKDEVICE_SIZE
```

Comment:
I don't think I can face changing them all in this branch...and you wouldn't want to review the changes. I'll create a followup issue if you agree.